packages/usa-combo-box/src/index.mjs

```diff
@@ -1,9 +1,9 @@
-const keymap = require("receptor/keymap");
-const selectOrMatches = require("../../uswds-core/src/js/utils/select-or-matches");
-const behavior = require("../../uswds-core/src/js/utils/behavior");
-const Sanitizer = require("../../uswds-core/src/js/utils/sanitizer");
-const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
-const { CLICK } = require("../../uswds-core/src/js/events");
+import keymap from "receptor/keymap/index.js";
```

Comment:
**TODO: Investigate further**

The component worked with importing keymap without extension. Updated to this path because mocha couldn't find the import.